common/culture/cultures/wc_azsharic.txt

```diff
@@ -18,7 +18,7 @@
 	
 	coa_gfx = { western_coa_gfx wc_pagan_coa_gfx }
 	building_gfx = { mediterranean_building_gfx }
-	clothing_gfx = { creature_naga_gfx high_elven_clothing_gfx high_elven_hairstyles_gfx no_headgear_gfx }
```

Comment:
I belive `no_headgear_gfx` wasn't removed, it was replaced by `no_lowborn_headgear_gfx`.
The logic behind ogres, elves, demons, draenei prefering no headgear is ears, horns cliping with headgears.

Reply:
I added `has_trait_with_flag = dragon_family` to the following trigger. Do you prefer I use `has_clothing_gfx = no_lowborn_headgear_gfx` instead? It would not be consistent with the other exceptions listed below.
```
prefer_no_lowborn_headgear = {
	OR = {
		# Ears get in the way
		culture = { has_clothing_gfx = no_lowborn_headgear_gfx }
		has_trait_with_flag = elven_family
		has_trait_with_flag = dragon_family
		is_race_no_gene_trigger = { RACE = creature_troll }
		is_race_no_gene_trigger = { RACE = creature_goblin }
		is_race_no_gene_trigger = { RACE = creature_ogre }
		is_race_no_gene_trigger = { RACE = creature_draenei }
		is_race_no_gene_trigger = { RACE = creature_sayaadi }
	}
}
```